Makefile

```diff
@@ -19,18 +19,38 @@ images_set_variables: images_check_env
 	$(eval docker_build_tag := "buildtag_$(BUILD_NUMBER)")
 	$(eval tagged_image_list := $(shell docker images | grep "$(docker_build_tag)" | cut -d" " -f1 | cat))
 	$(eval git_tag_for_current_branch := $(shell git tag --points-at)) #will be non-empty if HEAD is tagged
+	$(eval docker_networks := $(shell docker network inspect amazeeio-network | grep -o '\"Name\": \"[^\"]*' | sed 's/^.*: //' | sed 's/"//g' | cat))
 
 # This target is simply a test of the above
 .PHONY: images_echo_variables
 images_echo_variables: images_set_variables
 	echo $(docker_build_tag)
 	echo $(tagged_image_list)
 	echo $(git_tag_for_current_branch)
+	echo $(docker_networks)
 
 # This target will build out the images, passing the correct environment vars to fill out repo and tags
 .PHONY: images_build
 images_build: images_set_variables
-	DOCKER_REPO=$$DOCKER_REPO BUILDTAG=$(docker_build_tag) docker-compose build
+	HAS_IO_NETWORK=false; \
```

Comment:
I think this should go in its own target and then make `images_build` call it as a prerequisite.